manifests/params.pp

```diff
@@ -53,7 +53,21 @@
     }
 
     default: {
-      fail "Unsupported OS family '${::osfamily}'"
+      case $::operatingsystem {
```

Comment:
Is there no `$::osfamily ` for Alpine that would avoid having to do a new case inside?